Formula/kubernetes-cli.rb

```diff
@@ -48,8 +55,10 @@ def install
     run_output = shell_output("#{bin}/kubectl 2>&1")
     assert_match "kubectl controls the Kubernetes cluster manager.", run_output
 
-    version_output = shell_output("#{bin}/kubectl version --client 2>&1")
-    assert_match "GitTreeState:\"clean\"", version_output
+    # Due to ARM patch reinstate for next release
+    #version_output = shell_output("#{bin}/kubectl version --client 2>&1")
+    #assert_match "GitTreeState:\"clean\"", version_output
```

Comment:
Can we reference the test in the comment above the patch, so that this isn't forgotten when the patch is removed?